pkg/scope/cluster.go

```diff
@@ -150,9 +152,25 @@ func (s *ClusterScope) setupProxmoxClient(ctx context.Context) (capmox.Client, e
 	tokenSecret := string(secret.Data["secret"])
 	url := string(secret.Data["url"])
 
-	// TODO, check if we need to delete tls config
+	tlsInsecure := string(secret.Data["insecure"]) != "false"
```

Comment:
Cast to boolean? `secret.Data` is `map[string][]byte`… …the only other way would be `strconv.ParseBool` and that's basically checking for `true` vs `false`.

Using `""` as `false` would be a breaking change as the default would shift to secure connections only. (`""` will be used by everyone using a previous version as that parameter currently does not exist.)

I decided to go with explicit `false` to prevent introducing a breaking change.